server/pulp/plugins/util/saxwriter.py

```diff
@@ -0,0 +1,94 @@
+from xml.sax.handler import ContentHandler
+from xml.sax.saxutils import escape, quoteattr
+
+
+class XMLWriter(ContentHandler):
```

Comment:
Is some or all of this backported from python 3.y? If so, it would be a good idea to state that in comments both on the class, and on individual methods.